systemvm/patches/debian/config/opt/cloud/bin/configure.py

```diff
@@ -479,13 +479,13 @@ def process(self):
 
     def deletevpn(self, ip):
         logging.info("Removing VPN configuration for %s", ip)
-        CsHelper.execute("ipsec auto --down vpn-%s" % ip)
-        CsHelper.execute("ipsec auto --delete vpn-%s" % ip)
+        CsHelper.execute("ipsec down vpn-%s" % ip)
+        CsHelper.execute("ipsec down vpn-%s" % ip)
```

Comment:
@terbolous 
Doing twice will  clean up the tunnel/session. Doing once I have observed failures in  bringing up the tunnel some times. So doing it twice for safe side.